### DISCUSSION TITLE
Apple Silicon M1 Mac - Is it possible to boot from PXE?

### DISCUSSION THREAD

**Discussion by nyp-jeremyt** [ipxe/ipxe#837] [Q&A]
Hi,
I have finally decided to embark on this journey to explore the possibilities of imaging an Apple Silicon M1 Mac.

The challenges I have encountered is the booting into PXE, is this possible or in kind of a development to try this out?

**Comment by NiKiZe** [comment]
Similar question in the FOG Project recently posted, was that you? https://forums.fogproject.org/topic/16627

iPXE can be built for ARM.
If you build snponly.efi for ARM64 then in theory it should only be a matter of running that binary?

If you can get grub to work then replace or use iPXE from there?

**Comment by stappersg** [comment]
> to embark on this journey to explore

Please sent postcards or whatever travellers do share with _"home front"_ these days. I do love to read stories about escapes and jail breaking.  :smiley:  I'm fully aware that [`Apple Silicon M1 Mac boot methods`](https://duckduckgo.com/?t=ffab&q=Apple+Silicon+M1+Mac+boot+methods&ia=web) is not an _iPXE_ thing, but it is the netboot part I do like to know more about.

**Reply by stappersg** [reply]
[link to #906](https://github.com/ipxe/ipxe/issues/906)
